conseil-lorre/src/test/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinOperationsTest.scala

```diff
@@ -32,8 +31,70 @@ class BitcoinOperationsTest
   implicit val contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
 
   "Bitcoin operations" should {
-      // "..." in new BitcoinPersistenceStubs(dbHandler) with BitcoinOperationsStubs { 
-      // }
+      "run indexer for the newest blocks" in new BitcoinOperationsStubs {
+        // check if the method gets the latest block at the beginning
+        (bitcoinPersistenceMock.getLatestIndexedBlock _) expects ()
+        (txMock.transact[Option[Tables.BlocksRow]] _) expects (*) returning (IO(
+              Some(DbFixtures.blockRow.copy(height = 5))
+            ))
+
+        // mock Bitcoin client calls
+        (bitcoinClientMock.getBlockChainInfo _) expects () returning (Stream(BlockchainInfo("mainnet", 10)).covary[IO])
+
+        // test if method runs indexer with the correct range of the blocks
+        (dummyBitcoinOperationsMock.loadBlocksWithTransactions _) expects (6 to 10) returning (Stream.empty)
+
+        dummyBitcoinOperationsStub.loadBlocks(Newest).compile.drain.unsafeRunSync() shouldBe (())
```

Comment:
Is there any real need to compare the unit result to something?

Am I right in my understanding that the test is supposed to verify what internals are being called?

Reply:
Maybe you're right, I've removed it